src/agent/debuglet.ts

```diff
@@ -242,6 +243,15 @@ export class Debuglet extends EventEmitter {
     this.breakpointFetched = null;
     this.breakpointFetchedTimestamp = -Infinity;
     this.debuggeeRegistered = new CachedPromise();
+
+    const that = this;
+    function debugAgentWarningListener(warning: NodeJS.ErrnoException) {
```

Comment:
Do we still need this? The issue has been fixed in Node at this point, and we will no longer be getting this warning, correct?

Reply:
Yes, I forgot about that.  I'm closing this in favor of PR #376